Dockerfile

```diff
@@ -15,10 +15,11 @@ ENTRYPOINT poetry run flask run -h 0.0.0.0 -p 5000
 # production build stage
 FROM base as production
 ENV FLASK_ENV=production
+ENV PORT=33507
```

Comment:
This default port means your README instructions for your prod container won't work by default - they expose port 5000 but the app ends up running on 33507. We should either adjust this default, or update those instructions

Reply:
Changed in Readme